pkg/ccl/changefeedccl/kvfeed/scanner.go

```diff
@@ -72,6 +72,13 @@ func (p *scanRequestScanner) Scan(
 	var atomicFinished int64
 	for _, span := range spans {
 		span := span
+
+		currentMaxConcurrentExports := maxConcurrentExportRequests(p.gossip, &p.settings.SV)
+		if currentMaxConcurrentExports != maxConcurrentExports {
```

Comment:
Another option here would've been to make `SetLimit` a no-op when setting it to the current limit (right now regardless of value it'll unlock/lock a mutex to make the change).  Then this could just call `SetLimit` on every run without a conditional.  I just wasn't sure if anything would go wrong with doing that.

Reply:
I would actually only recompute max if the cluster setting changed.  We currently have logic that counts number of nodes in the cluster...  I don't know if we want to necessarily react to transient cluster state changes.  Plus having hot loops might not be best anyway.